lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -593,6 +677,30 @@ describe('NxCombobox', function() {
           expect(multiError.getByRole('combobox')).not.toHaveAttribute('aria-invalid', 'true');
         });
 
+        describe('when in a form with showValidationErrors', function() {
+          function quickRender(extraProps?: Partial<Props>) {
+            const renderResult = render(
+              <NxForm showValidationErrors onSubmit={() => {}}>
+                <NxCombobox data-testid="combobox" { ...nonValidatableMinimalProps } { ...extraProps } />
```

Comment:
does the data-testid need to be here, same as https://github.com/sonatype/sonatype-react-shared-components/pull/789/files#r976899751?

Reply:
> does the data-testid still need to be here?

No. https://github.com/sonatype/sonatype-react-shared-components/commit/98bd2530253ee6eb992af4a172d2c524ed0e8c85